hm-service-account-api-rest-v1.yml

```diff
@@ -355,6 +355,61 @@ paths:
       description: >-
         Get the status of a VIN that has previously been registered for data
         access clearance.
+    delete:
+      security:
+        - ServiceAccountToken: []
+      tags:
+        - Fleet
+      description: >-
+        Delete clerance for the given VIN. If the clearance is in pending state, it cancel the clearance otherwise tries to revoke the clearance.
```

Comment:
```suggestion
        Delete clearance for the given VIN. If the clearance is in pending state, it cancel the clearance otherwise tries to revoke the clearance.
```

Reply:
`Delete the clearance for the given VIN. If the clearance is in a pending state, the activation process is canceled. If the vehicle is in an approved state, a revoke is attempted.`